### ISSUE TITLE
So, this is not OK?

### ISSUE THREAD

**Issue by imcodybell** [daviddwlee84/FRP-Heroku#6]
currently, heroku only support 80/443.

but, we can/may use reverse proxy to resolved(must need 1 persnal domain):

1 app:
- 1 domain -> reverse proxy -> 1 port
- another 1 domain -> reverse proxy -> another 1 other
- ......

**Comment by daviddwlee84** [comment]
Yeah, we need another normal server (personal domain) to reverse proxy the reverse proxy. This makes the Heroku server meaningless.
